lib/driver.dart

```diff
@@ -0,0 +1,407 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert' show json;
+import 'dart:html' as html;
+import 'dart:math' as math;
+
+import 'package:web_benchmarks/recorder.dart';
+
+typedef RecorderFactory = Recorder Function();
+
+const bool isCanvasKit = bool.fromEnvironment('FLUTTER_WEB_USE_SKIA', defaultValue: false);
```

Comment:
This is not needed here. Also, it depends on the internal `FLUTTER_WEB_USE_SKIA` flag which can change any time.

Reply:
Done.